models/intel/handwritten-simplified-chinese-recognition-0001/description/handwritten-simplified-chinese-recognition-0001.md

```diff
@@ -0,0 +1,46 @@
+# handwritten-simplified-chinese-recognition-0001
+
+## Use Case and High-Level Description
+
+This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
+reshape layer and a fully connected layer.
+The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).
+
+## Example
+
+![](./test.png) -> 的人不一了是他有为在责新中任自之我们
+
+## Specification
+
+| Metric                                         | Value              |
+|------------------------------------------------|--------------------|
+| GFlops                                         |                    |
+| MParams                                        |                    |
+| Accuracy on partial SCUT-EPT test set          |   75.58%           |
+| Source framework                               | PyTorch\*          |
+
+
+## Accuracy Values
+
+This demo adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.
+
+## Inputs
+Shape: [1x1x96x2000] - An input image in the format [BxCxHxW],
+where:
+  - B - batch size
+  - C - number of channels
+  - H - image height
+  - W - image width
+
+Note that the source image will be converted to grayscale, resized to specific height (such as 96) while keeping aspect ratio, and right-bottom padded.
```

Comment:
1. "Grayscale" is moved above.
2. If you really need to emphasize the importance of resize options (also you miss the fill value), make it a real note:
> **NOTE:** Image must be resized ...
```
> **NOTE:** Image must be resized ...
```
In other case - delete this sentence completely.